### PR TITLE
Push signature method to manage for OIDC

### PR DESCRIPTION
**Pull request by pablothedude**
When pushing an OIDC entity to manage the signature method
wasn't pushed.

https://www.pivotaltracker.com/n/projects/1400064/stories/166838510

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php [hunk 1]
```diff
@@ -238,8 +238,10 @@ private function generateMetadataFields(Entity $entity)
             $metadata['AssertionConsumerService:0:Binding'] = $entity->getAcsBinding();
             $metadata['AssertionConsumerService:0:Location'] = $entity->getAcsLocation();
             $metadata['NameIDFormat'] = $entity->getNameIdFormat();
+            $metadata['coin:signature_method'] = 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256';
             $metadata = array_merge($metadata, $this->generateSecurityMetadata($entity));
         } else if ($entity->getProtocol() == Entity::TYPE_OPENID_CONNECT) {
+            $metadata['coin:signature_method'] = 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256';
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php]: I think you can get away with calling the `generateSecurityMetadata` method for both protocol types. Added benefit of that is that the setting of the signature method remains 'responsibility' of that method.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php [hunk 2]
```diff
@@ -25,7 +25,6 @@ class MetaData
     /**
      * Supports 1 ACS location, the first entry is used that is passed from Manage
      */
-    private $acsBinding;
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php]: Please explain why you are removing the acsBinding field from the metadata DTO. I see it is not used. But seeing this removed in this PR deserves at least a mention in the PR description or your commit message.

File: src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php (hunk 1 above)
Comment (pablothedude) [reply src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php]: The `certData` is omitted in OIDC so therefore I don't want to push it to manage. However renamed the method to `generateSecurityMetadata` to reflect it's responsibility better.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php (hunk 2 above)
Comment (pablothedude) [reply src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php]: Mentioned it in the commit message.

File: src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php (hunk 1 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php]: This is fine too. By calling the method you would have saved a couple of lines of code.

**Pull request merged**